src/DiskIO/IpcIo/IpcIoFile.h

```diff
@@ -99,7 +99,7 @@ class IpcIoFile: public DiskFile
 
 protected:
     friend class IpcIoPendingRequest;
-    void openCompleted(const Ipc::StrandSearchResponse *const response);
+    void openCompleted(const Ipc::StrandMessage *);
```

Comment:
Dropped a const from the member signature here.

```suggestion
    void openCompleted(const Ipc::StrandMessage * const);
```

Reply:
I dropped this const because it is ignored by the compiler. @rousskov suggests removing 'const' in such cases. Here is also an [article](https://abseil.io/tips/109) favoring this approach.

Reply:
Just to add one more bit of supporting information to Eduard's "this polishing is the right thing to do" response: IIRC, when @kinkie enabled more useful compiler checks, these "extra" `const` in function _declarations_ started to produce false positives. If really needed, I can try to find the corresponding errors/discussions in the long history of #667.

@yadij, please reconsider your change request.

@eduard-bagdasaryan, great [ToW](https://abseil.io/tips/109) find! I will be adding its rules of thumb to our squid-notes.

Reply:
> I will be adding its rules of thumb to our squid-notes.

Actually, this rule of thumb was already documented as [#style-const](https://github.com/measurement-factory/squid-notes/blob/start/thumbs.md#style-const). I now added a supporting ToW reference.